pkg/database/gorm.go

```diff
@@ -47,22 +33,29 @@ func getDialectorFromConfig(config *Config, environment string) gorm.Dialector {
 		connectionString = testDriverName
 	}
 
-	return mysql.New(mysql.Config{
-		DSN:        connectionString,
-		DriverName: driverName,
-	})
-}
+	dbServiceName := fmt.Sprintf("%s-mysql", appName)
+	driverServiceName := fmt.Sprintf("%s-orm", appName)
```

Comment:
Current value for `dbServiceName` is with `-mysql`, this way we will have `document-search-orm`.
Maybe I change `driverServiceName` to `appname-dbdriver`?

Reply:
I agree with @fotos about the naming: the `-orm` should be assigned to the `gorm` instrumentation, and the `-mysql` to the MySQL driver instrumentation.

@laynax If we change the name for the gORM instrumentation, what happens? I think the only thing will change is the DataDog metric names, which we will need to update anyway

Reply:
@fotos @Neurostep 

Changed as requested 👍 

Overall I'm against having more than one service name for each service, it must be the operation name providing info about `orm` or `mysql` type.

Reply:
@fotos A brief update here. After discussing this PR with @laynax we decided to keep the service name as is `%s-mysql` (`document-search-mysql`) as it will make things easier and effectively a part of one `database` flow. One can distinguish between different span types using the operation type drop-down (see screenshot). Please, let us know what you think about this 🙏 


![Screenshot 2024-06-28 at 10 34 05](https://github.com/scribd/go-sdk/assets/292940/1cfcce90-596a-424b-83c2-0a9aac336e3d)

@laynax Let's update the service name here using `-mysql` suffix for both clients 👍

Reply:
Sounds good to me!

Let's see how that works in practice. 👍